tracker/views.py

```diff
@@ -0,0 +1,44 @@
+from django.shortcuts import render,redirect
+from .models import * 
+from .forms import * 
+
+# Create your views here.
+# This method (1)
+def home(request):
+    if request.method == "POST":
+        name = request.POST['name']
```

Comment:
Method 2

Reply:
When I use the Modelform, the rendered is a little different. I get a text field for the Timefield that's why I opted for the realization of the form myself. Otherwise, in the code I introduced both methods.

Reply:
It is definitely better with Template view and I think you can do even simpler by using a class based view with mixins for List (MultipleObjectMixin) and Create. https://docs.djangoproject.com/fr/3.1/topics/class-based-views/mixins/#avoid-anything-more-complex. You can try and tell me if it working and not why

Reply:
Can you use a form rather than using direct calls to POST?
A form will help you to check that the data have the good format.